app/config/settings.py

```diff
@@ -477,6 +477,7 @@ def strtobool(val) -> bool:
     "grandchallenge.datatables",
     "grandchallenge.organizations",
     "grandchallenge.groups",
+    "grandchallenge.webhooks",
```

Comment:
What do you think about calling the app `github`? Webhooks might be a bit too generic and the settings are already namespaced with `GITHUB_`.

Reply:
That's probably better, as there's also a redirect view in there now. Not just a webhook receiver.